.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - run: sudo apt update
+    - run: sudo apt install libpng-dev libldap2-dev php7.2-zip php7.2-bcmath php7.2-gd php7.2-ldap
+    - name: Update Composer
+      run: sudo composer self-update
+    - name: Cache vendor directory
+      uses: actions/cache@preview
+      with:
+        path: vendor
+        key: ${{ runner.os }}-composer
+    - name: Install Composer dependencies
+      run: composer install --no-interaction --no-progress --no-suggest --no-dev
+      env:
+        COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}
+    - name: Check for env() being used outside config/
+      run: bash -c "cd $GIT_WORKSPACE && ! git grep \"env(\" -- '.' ':!/config/' ':!/.circleci/config.yml' ':!/composer.lock'"
+    - run: php artisan config:cache
+    - run: php artisan nova:publish --no-interaction
+    - run: php artisan horizon:assets --no-interaction
+    - name: Cache npm
+      uses: actions/cache@preview
+      with:
+        path: ~/.npm
```

Comment:
What is in here?

Reply:
That's npm's cache of downloads from the registry.